internal/csv/csv.go

```diff
@@ -8,61 +8,50 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/operator-framework/api/pkg/operators/v1alpha1"
 	"github.com/pkg/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-// NewClusterServiceVersion returns a new *ClusterServiceVersion with the YAML in
+// OverrideClusterServiceVersion returns a new *ClusterServiceVersion with the YAML in
 // outputDir if it exists, otherwise it will return an empty *ClusterServiceVersion.
```

Comment:
I think this line may need to be updated as it doesn't look like we will create a new cluster service version.

Reply:
Nice catch, thanks!